src/__tests__/postal-code.ts

```diff
@@ -41,6 +41,16 @@ describe("postalCode", () => {
       [["hello world", { isValid: true, isPotentiallyValid: true }]],
     ],
 
+    [
+      "returns isPotentiallyValid for strings without alphanumeric first three characters",
+      [
+        ["123", { isValid: true, isPotentiallyValid: true }],
+        ["  123", { isValid: true, isPotentiallyValid: true }],
+        ["---", { isValid: false, isPotentiallyValid: true }],
```

Comment:
@jplukarski logically you would think that, but `isPotentiallyValid` just checks if the data type is a string or not.

Reply:
Right, because before we were only concerned with the length of the postal code, but now we are adding this check for alphanumeric characters, so shouldn't we return false in these cases now since if there is a non-alphanumeric character present then the postal code could not be valid?

Reply:
After offline discussion, I think it makes sense to leave this as is since theoretically a user could enter alpha-numeric characters inbetween the `-`. So it makes sense to return `true` here